matchya-web/src/components/AuthModal.tsx

```diff
@@ -0,0 +1,79 @@
+import FormInput from "./FormInput";
+import Button from "./Button";
+import matchIcon from "../assets/matchya-icon.png";
+import { useState } from "react";
```

Comment:
Let's sort the imports so it takes the order
1. Built-in libraries
2. Third-party libraries (ex.react)
3. Relative imports